wire-library/wire-protoc/src/main/java/com/squareup/wire/protocwire/Plugin.java

```diff
@@ -210,9 +210,28 @@ public static void run(CodeGenerator generator, Environment environment) {
 
     CodedOutputStream output = CodedOutputStream.newInstance(
       environment.getOutputStream());
+    try {
+      // go ahead and write response preamble
+      CodeGeneratorResponse.newBuilder()
+        // add more here as more features are introduced and then supported
+        .setSupportedFeatures(
+          toFeatureBitmask(CodeGeneratorResponse.Feature.FEATURE_PROTO3_OPTIONAL))
```

Comment:
I know we haven't actually implemented support for proto3 optionals...but we will!